### DISCUSSION TITLE
Showing more error info

### DISCUSSION THREAD

**Discussion by amacfie-tc** [dbcli/pgcli#1407] [Q&A] [answered]
By default when an error occurs, not all fields of the error are shown. In psql we can do [this](https://dba.stackexchange.com/a/28802/249985) to view them all (SQLSTATE, TABLE NAME, etc.). Does pgcli have a way to do this?

**Comment by j-bennet** [comment]
pgli does not have `\set`, but you can do this:

```
set log_error_verbosity ='verbose'
```

**Reply by amacfie-tc** [reply]
Thanks, although that doesn't appear to give as much info as psql.

**Reply by andrew222651** [reply, marked as answer]
PR opened https://github.com/dbcli/pgcli/pull/1455
